R/tt_sort.R

```diff
@@ -50,29 +57,107 @@ cont_n_onecol <- function(j) {
     }
 }
 
-#' Sort substructure of a TableTree at a particular Path in the Tree.
+#' Sorting a Table at a Specific Path
+#' 
+#' @description Main sorting function to order the substructure of a `TableTree` 
+#'   at a particular Path in the table tree.
+#' 
 #' @inheritParams gen_args
 #' @param scorefun function. Scoring function, should accept the type of
-#'   children directly under the position at \code{path} (either VTableTree,
-#'   VTableRow, or VTableNodeInfo, which covers both) and return a numeric value
+#'   children directly under the position at \code{path} (either `VTableTree`,
+#'   `VTableRow`, or `VTableNodeInfo`, which covers both) and return a numeric value
 #'   to be sorted.
 #' @param decreasing logical(1). Should the the scores generated by
 #'   \code{scorefun} be sorted in decreasing order. If unset (the default of
 #'   \code{NA}), it is set to \code{TRUE} if the generated scores are numeric
 #'   and \code{FALSE} if they are characters.
-#' @param na.pos character(1). What should be done with children (subtrees/rows)
+#' @param na.pos character(1). What should be done with children (sub-trees/rows)
 #'   with \code{NA} scores. Defaults to \code{"omit"}, which removes them, other
 #'   allowed values are \code{"last"}  and \code{"first"} which indicate where
 #'   they should be placed in the order.
 #' @param .prev_path character. Internal detail, do not set manually.
-#' @return A TableTree with the same structure as \code{tt} with the exception
+#' 
+#' @return A `TableTree` with the same structure as \code{tt} with the exception
 #'   that the requested sorting has been done at \code{path}
-#' @details The \code{path} here can include \code{"*"} as a step, which means
-#' that each child at that step will be \emph{separately} sorted based on
-#' \code{scorefun} and the remaining \code{path} entries. This can occur
-#' multiple times in a path.
-#'
-#'
+#'   
+#' @details The \code{path} here can include the "wildcard" \code{"*"} as a step, 
+#'   which translates roughly as *any* node/branching element and means
+#'   that each child at that step will be \emph{separately} sorted based on
+#'   \code{scorefun} and the remaining \code{path} entries. This can occur
+#'   multiple times in a path. 
```

Comment:
```suggestion
#' @details The \code{path} here can include the "wildcard" \code{"*"} as a step, 
#'   which translates roughly to *any* node/branching element and means
#'   that each child at that step will be \emph{separately} sorted based on
#'   \code{scorefun} and the remaining \code{path} entries. This can occur
#'   multiple times in a path. 
```